server/controllers/groups.js

```diff
@@ -552,70 +530,22 @@ module.exports = function(app) {
    * Get group content.
    */
   const getOne = (req, res, next) => {
-    async.auto({
-
-      getBalance: getBalance.bind(this, req.group.id),
-      getYearlyIncome: getYearlyIncome.bind(this, req.group.id),
-      getPublicPageInfo: getPublicPageInfo.bind(this, req.group.id),
-
-      getActivities: (cb) => {
-        if (!req.query.activities && !req.body.activities) {
-          return cb();
-        }
-
-        const query = {
-          where: {
-            GroupId: req.group.id
-          },
-          order: [['createdAt', 'DESC']],
-          offset: 0,
-          limit: 20 // [TODO] I need to put this default value
-          // as a global parameter. Using mw.paginate?
-        };
-
-        Activity
-          .findAndCountAll(query)
-          .then((activities) => cb(null, activities.rows))
-          .catch(cb);
-      },
-
-      getStripeAccount: (cb) =>  {
-        req.group.getStripeAccount()
-          .then((account) => cb(null, account))
-          .catch(cb);
-      },
-
-      getConnectedAccount: (cb) => {
-        req.group.getConnectedAccount()
-          .then((account) => cb(null, account))
-          .catch(cb);
-      }
-
-    }, (e, results) => {
-      if (e) return next(e);
-
-      const group = _.extend({}, req.group.info, {
-        balance: results.getBalance,
-        yearlyIncome: results.getYearlyIncome,
-        backersCount: results.getPublicPageInfo.backersCount,
-        donationTotal: results.getPublicPageInfo.donationTotal
-      });
-
-      if (results.getActivities) {
-        group.activities = results.getActivities;
-      }
-
-      if (results.getStripeAccount) {
-        group.stripeAccount = _.pick(results.getStripeAccount, 'stripePublishableKey');
-      }
-
-      if (_.get(results, 'getConnectedAccount.provider') === 'paypal') {
-        group.hasPaypal = true; // hack for the prototype, to refactor
-      }
-
-      res.send(group);
-    });
-
+    const group = req.group.info;
+
+    req.group.getStripeAccount()
+      .tap(account => group.stripeAccount = account && _.pick(account, 'stripePublishableKey'))
+      .then(() => req.group.getConnectedAccount())
```

Comment:
both should work, because `getConnectedAccount ` takes no param. But yes I prefer `.then(req.group.getConnectedAccount)`.